README.md

```diff
@@ -479,15 +479,7 @@ Reference: https://github.com/facebook/jest/issues/708
 
 ### Configure other JSDOM versions
 
-**Jest** v25 by default uses **JSDOM** 15 to support Node 8, which should also be compatible with Jest 24 and earlier.
-
-If you need a newer JSDOM than the one that ships with Jest and run Node >=10, you can install `jest-environment-jsdom-sixteen` package, and edit your Jest config like so:
-
-```
-{
-  "testEnvironment": "jest-environment-jsdom-sixteen"
-}
-```
+**Jest** v26 by default uses **JSDOM** 16 to support Node 10+.
```

Comment:
Is JSDOM 16 compatible with Jest 25 ? I'm not sure what to write here. Any thoughts ?

Reply:
It should be. But in 99% of the cases, whoever uses Jest v26 will be happy to use JSDOMv16 and Nodev10.
JSDOM is not a interesting test-target like IE11 or Edge, so a specific setup will not be wanted by 99% of jest users.

I think we can remove that "should also be compatible" part like you did 👍

Reply:
`jest-environment-jsdom-sixteen ` et. al. are compatible with any version of Jest. Knowing jsdom project, they'll drop Node 10 way earlier than Jest does, so this section may still be handy. You could provide references to specific tags, so people on lower versions can refer to

Reply:
I brought back the part for providing references to specific tags as suggestion :)